tensorflow/core/common_runtime/gpu/process_state.cc

```diff
@@ -181,11 +181,23 @@ Allocator* ProcessState::GetCUDAHostAllocator(int numa_node) {
   // different numa_nodes.  For now, just one.
   numa_node = 0;
   mutex_lock lock(mu_);
+
+  // CUDAHost alloc the same across all gpus, so just get the executor for the
+  // first valid device. We can't just use 0 because 0 may not be a valid device
+  // in this tensorflow session.
+  gpu::Platform* gpu_platform = GPUMachineManager();
+  gpu::StreamExecutor* se = nullptr;
+  for (int gpu_id = 0;
```

Comment:
If you could, try running clang-format --style=google on this file, it should automatically fix up spacing and indentation to conform to existing style.